src/main/java/org/wise/portal/presentation/web/filters/WISEAuthenticationFailureHandler.java

```diff
@@ -89,16 +89,8 @@ public void onAuthenticationFailure(HttpServletRequest request, HttpServletRespo
       try {
         responseJSON.put("status", "failure");
         boolean isReCaptchaRequired = ControllerUtil.isReCaptchaRequired(request);
-        if (isReCaptchaRequired) {
-          responseJSON.put("requireRecaptcha", true);
-          String contextPath = request.getContextPath();
-          response.sendRedirect(contextPath + "/login/recaptcha");
-        } else {
-          String contextPath = request.getContextPath();
-          response.sendRedirect(contextPath + "/login");
-        }
+        responseJSON.put("isRecaptchaRequired", isReCaptchaRequired);
```

Comment:
```suggestion
        responseJSON.put("isRecaptchaRequired", ControllerUtil.isReCaptchaRequired(request));
```
and remove line above.